### PR TITLE
Make it possible to customize local journal mappings

### PR DESCRIPTION
**Pull request by aignas**
A small addition of an extra setting and a short intro in the docs.

### CODE REVIEW

File: autoload/wiki/buffer.vim [hunk 1]
```diff
@@ -175,6 +175,9 @@ function! s:init_buffer_mappings() abort " {{{1
   endif
 
   call extend(l:mappings, get(g:, 'wiki_mappings_local', {}))
+  if b:wiki.in_journal
+    call extend(l:mappings, get(g:, 'wiki_mappings_journal_local', {}))
```
Comment (lervag) [inline autoload/wiki/buffer.vim:179]: Could I suggest `local_journal` instead of `journal_local`?

File: doc/wiki.txt [hunk 2]
```diff
@@ -616,6 +617,9 @@ OPTIONS                                                   *wiki-config-options*
   visual mode, respectively. The available `<plug>` mappings are listed in
   |wiki-mappings|.
 
+  For any local mappings that should only be applied in journal entries,
+  consider setting |g:wiki_mappings_journal_local|.
+
```
Comment (lervag) [inline doc/wiki.txt:622]: Perhaps one should add an example for this as well?

**Review by lervag — CHANGES_REQUESTED** [review]
Thanks! This looks like a useful addition. I have two minor comments.

File: doc/wiki.txt (hunk 2 above)
Comment (aignas) [reply doc/wiki.txt:631]: Added an example and an extra table for the defaults.

File: autoload/wiki/buffer.vim (hunk 1 above)
Comment (aignas) [reply autoload/wiki/buffer.vim]: Done

**Comment by lervag** [comment]
Thanks! I hope you don't mind that I made a minor adjustment on top of this. In any case, I appreciate your contribution!

**Comment by aignas** [comment]
> Thanks! I hope you don't mind that I made a minor adjustment on top of this. In any case, I appreciate your contribution!

Thanks! no worries. My main goal was to get the setting in, so I am happy that you could review this this quickly!

**Pull request merged**